src/sota_tools/authenticate_test.cc

```diff
@@ -11,9 +11,12 @@
 #include "treehub_server.h"
 #include "utilities/utils.h"
 
+boost::filesystem::path certs_dir;
+
 /* Authenticate with OAuth2.
  * Parse authentication information from treehub.json. */
 TEST(authenticate, good_zip) {
+  // Authenticates with the ATS portal to the SaaS instance.
```

Comment:
What is "the ATS portal", is it connect.ota.here.com  ?

Reply:
Nope, that's the normal portal. https://app.atsgarage.com/ is the old portal that surprisingly still works but basically serves the same content.